apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -298,6 +325,29 @@ export class TLDrawDurableObject {
 		const { 0: clientWebSocket, 1: serverWebSocket } = new WebSocketPair()
 		serverWebSocket.accept()
 
+		const closeSocket = (code: number, message: string) => {
+			serverWebSocket.close(code, message)
+			return new Response(null, { status: 101, webSocket: clientWebSocket })
+		}
+
+		if (this.isApp) {
+			const ownerId = await this.getOwnerId()
+
+			if (ownerId) {
+				const auth = await requireAuth(req, this.env)
+				if (ownerId !== auth.userId) {
```

Comment:
ach, i was afraid i'd get to this point in the code and see this heh :)

i'm gonna be pendantic here and say that the wording here shouldn't be `NOT_AUTHORIZED` but should be `FORBIDDEN`.

it's the 401 vs. 403 distinction. it would be good to keep the lingo on this consistent with the HTTP wording because otherwise people will trip up on this distinction.